README.md

```diff
@@ -109,6 +119,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 *Libraries related to handling date and time.*
 
+* [JDateTime](http://jodd.org/doc/jdatetime.html) - Developer-friendly, astronomically precise date/time class based on Julian dates.
```

Comment:
I really can't see the reason to prefer JDateTime over Joda-Time or Java 8. Joda-Time has a convenient API which encapsulates the complexity and provides a rather simple interface. It also provides the featue of the edge case of Julian dates.